uw-research-computing/gpu-jobs.md

```diff
@@ -112,9 +112,10 @@ For those who would like to pursue alternative GPU resources, see our list of
     <th>Number of Servers</th>
     <th>Names</th>
     <th>GPUs / Server</th>
-    <th>GPU Type</th>
-    <th>CUDACapability</th>
-    <th>HasCHTCStaging</th>
+    <th>GPU Type(<code>CUDADeviceName</code>)</th>
```

Comment:
```suggestion
    <th>GPU Type (<code>CUDADeviceName</code>)</th>
```

One more minor spacing change to help with the line wrapping.  Everything else looks good to me.